src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -848,4 +848,31 @@
           }
         }
       }
+    },
+    "/restart": {
```

Comment:
I wonder if you should try to keep the top-level clear for future additions? Maybe something like `/redpanda/admin/restart`? Or perhaps that's overkill.

Reply:
I agree. We can also use a better name
`/redpanda/admin/restart_schema_store`